docs/tutorial/new_controller.rst

```diff
@@ -0,0 +1,106 @@
+
+Implementing a Controller
+-------------------------
+
+A controller closes the loop, a class derived from the abstract class
+:py:class:`~pymoskito.simulation_modules.Controller`.
+Its only function calculates the controller output,
+which acts as the actuating variable for the model.
+
+In this scenario, the controller is linear. 
+The linearization of the nonlinear model needs to be done beforehand
+and is given in this tutorial as:
+
+.. math::
+    
+    \boldsymbol{A} =
+    \begin{pmatrix}
+        0 & 0 & 1 & 0\\
+        0 & 0 & 0 & 1\\
+        0 & \frac{m^{2}l^{2}g}{z} & -\frac{JD}{z} & \frac{mld}{z}\\
+        0 & -\frac{(M+m)mlg}{z} & \frac{mlD}{z} & -\frac{(M+m)d}{z}\\
+    \end{pmatrix} 
+    \ \
+    \boldsymbol{B} =
+    \begin{pmatrix}
+        0\\
+        0\\
+        \frac{J}{z}\\
+        -\frac{ml}{z}\\
+    \end{pmatrix} 
+    \ \
+    \boldsymbol{C} =
+    \begin{pmatrix}
+        1 & 0 & 0 & 0\\
+    \end{pmatrix} 
+    
+.. math::
+    
+    z = (M+m)\cdot J - m^{2}l^{2}
+    
+The linear control law is given with the control gain :math:`\boldsymbol{K}`
+and the prefilter :math:`\boldsymbol{V}` as:
+
+.. math::
+    
+    u = -\boldsymbol{K} \boldsymbol{x} + \boldsymbol{V} \boldsymbol{y_d}
+
```

Comment:
That's where the theory ends and the fun begins. Maybe add a subheading to mark this point?

Reply:
I disagree, since it would be the only subheading